src/env.h

```diff
@@ -652,6 +652,7 @@ class Environment final : public MemoryRetainer {
   void RunDeserializeRequests();
   // Should be called before InitializeInspector()
   void InitializeDiagnostics();
+  void InitializeEbpf();
```

Comment:
We should try to avoid growing Node.js internal interfaces as much as possible. If we were to add a new API linked to an Environment, we should do it by adding the api to `EnvInst`. In this case I don't think the initialization belongs to the environment, because it's a "global" (only to be called once) thing, so I would move it to EnvList.